virttest/utils_libvirt/libvirt_vmxml.py

```diff
@@ -25,6 +29,26 @@ def set_vm_attrs(vmxml, vm_attrs):
     return vmxml
 
 
+def check_guest_xml(vm_name, pat_in_dumpxml, option='', status_error=False):
+    """
+    Check the given pattern in the vm dumpxml
+
+    :param vm_name: vm name
+    :param pat_in_dumpxml:  str, the pattern to search in dumpxml
+    :param status_error: True if expect not existing, otherwise False
+    :raises: TestFail if the result is not expected
+    """
+    ret_stdout = virsh.dumpxml(vm_name, extra=option).stdout.strip()
```

Comment:
change 
stdout -->stdout_text ?

Reply:
@chunfuwen , when I check def command() in virsh.py, I find below in Line 812,813, so I think here 'stdout' is ok to use. How do you think of?
```
        ret.stdout = ret.stdout_text
        ret.stderr = ret.stderr_text
```